ansible/playbooks/roles/upgrade/tasks/kubernetes.yml

```diff
@@ -30,16 +72,41 @@
           when:
             - inventory_hostname in groups.kubernetes_master[1:]
 
+        - name: Remove k8s upgrade state file for masters
+          file:
+            path: "{{ kubernetes.upgrade_master_state_file_path }}"
+            state: absent
+
     - name: Upgrade nodes
-      when: kubelet_version is version('v' + version, '<=')
+      when: >
+        upgrade_node_state_file_status.stat.exists or
+        initial_kubelet_version is version('v' + version, '<')
       block:
+        - name: Create k8s upgrade state file for nodes
+          copy:
+            dest: "{{ kubernetes.upgrade_node_state_file_path }}"
+            content: Upgrade started
+            mode: u=rw,g=r,o=
+
         - name: k8s | Upgrade node to v{{ version }}
           include_tasks: kubernetes/upgrade-node.yml
           when:
             - groups.kubernetes_node is defined
             - inventory_hostname in groups.kubernetes_node
 
+        - name: Remove k8s upgrade state file for nodes
+          file:
+            path: "{{ kubernetes.upgrade_node_state_file_path }}"
+            state: absent
+
+
```

Comment:
Unnecessary newline. Could you apply formatting to be sure there is no such places and additional whitespaces?

Reply:
Yes, I will fix it.